lib/exposes.js

```diff
@@ -423,6 +423,14 @@ class Climate extends Base {
         return this;
     }
 
+    withSwingMode(modes, access=a.ALL) {
+        assert(!this.endpoint, 'Cannot add feature after adding endpoint');
+        const allowed = ['off', 'on', 'vertical', 'horizontal', 'both'];
```

Comment:
Looking at the HA docs, there does not seem to be a limitation on the allowed values. I propose that we also don't put any limitations (https://www.home-assistant.io/integrations/climate.mqtt/)

Reply:
I use `Climate.withFanMode` as template. It is not limited by [home-assistant](https://developers.home-assistant.io/docs/core/entity/climate/#fan-modes) but is limited [here](https://github.com/Koenkk/zigbee-herdsman-converters/blob/852b871de63bc557a850435f6c5fcdd6602f4ca6/lib/exposes.js#L420).